java/client/src/org/openqa/selenium/devtools/profiler/Profiler.java

```diff
@@ -0,0 +1,163 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.devtools.profiler;
+
+import static org.openqa.selenium.devtools.ConverterFunctions.map;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
+import com.google.common.reflect.TypeToken;
+import java.util.List;
+import java.util.Optional;
+import org.openqa.selenium.Beta;
+import org.openqa.selenium.devtools.Command;
+import org.openqa.selenium.devtools.Event;
+import org.openqa.selenium.devtools.profiler.model.ConsoleProfileFinished;
+import org.openqa.selenium.devtools.profiler.model.ConsoleProfileStarted;
+import org.openqa.selenium.devtools.profiler.model.Profile;
+import org.openqa.selenium.devtools.profiler.model.ScriptCoverage;
+import org.openqa.selenium.devtools.profiler.model.ScriptTypeProfile;
+
+public class Profiler {
+
+  /**
+   * Disable Profiling
+   */
+  public static Command<Void> disable() {
+    return new Command<>("Profiler.disable", ImmutableMap.of());
+  }
+
+  /**
+   * Enable Profiling
+   */
+  public static Command<Void> enable() {
+    return new Command<>("Profiler.enable", ImmutableMap.of());
+  }
+
+  /**
+   * start Profiling process
+   */
+  public static Command<Void> start() {
+    return new Command<>("Profiler.start", ImmutableMap.of());
+  }
+
+  /**
+   * stop Profiling process
+   *
+   * @return {@link Profile} with sampled values
+   */
+  public static Command<Profile> stop() {
+    return new Command<>("Profiler.stop", ImmutableMap.of(), map("profile", Profile.class));
+  }
+
+  /**
+   * Collect coverage data for the current isolate. The coverage data may be incomplete due to garbage collection.
+   *
+   * @return Array {@link ScriptCoverage}
```

Comment:
The return type is wrong, and the comment doesn't provide additional information over the method's signature. In cases like that, it's better to just skip the javadoc comment entirely.